lib/wallet/migrations.js

```diff
@@ -162,6 +162,90 @@ class MigrateChangeAddress extends AbstractMigration {
   }
 }
 
+/**
+ * Migrate account for new lookahead entry.
+ * Applies to WalletDB v0
+ */
+
+class MigrateAccountLookahead extends AbstractMigration {
+  /**
+   * Create migration object.
+   * @param {WalletMigratorOptions}
+   */
+
+  constructor (options) {
+    super(options);
+
+    this.options = options;
+    this.logger = options.logger.context('account-lookahead-migration');
+    this.db = options.db;
+    this.ldb = options.ldb;
+  }
+
+  /**
+   * We always migrate account.
+   */
+
+  async check() {
+    return types.MIGRATE;
+  }
+
+  /**
+   * Actual migration
+   * @param {Batch} b
+   * @returns {Promise}
+   */
+
+  async migrate(b) {
+    const wids = await this.ldb.keys({
+      gte: layout.W.min(),
+      lte: layout.W.max(),
+      parse: key => layout.W.decode(key)[0]
+    });
+
+    for (const wid of wids) {
+      await this.migrateWallet(b, wid);
+    }
```

Comment:
Do we also need to bump the walletDB version? A migrated wallet run in an old version of hsd will crash with a scary error instead of a version mismatch:

```
EncodingError: Out of bounds read (offset=89).
    at BufferReader.readU32BE (/Users/matthewzipkin/Desktop/work/hsd/node_modules/bufio/lib/reader.js:256:10)
    at readKey (/Users/matthewzipkin/Desktop/work/hsd/lib/wallet/account.js:857:30)
    at Account.read (/Users/matthewzipkin/Desktop/work/hsd/lib/wallet/account.js:781:19)
...
```